sources/api/views/assistant/EventSuggestionListView.ts

```diff
@@ -0,0 +1,9 @@
+import { EventSuggestionView} from "./EventSuggestionView";
+import { EventSuggestion } from "../../../utils/types/EventSuggestion";
+
+export class EventSuggestionListView {
```

Comment:
No need to create a dedicated file (and if we did in the past, we should not have)